fabric-ca-client/lib/helper.js

```diff
@@ -28,23 +28,27 @@ function checkRegistrar(registrar) {
 	if (!registrar.getSigningIdentity()) {
 		throw new Error('Can not get signingIdentity from registrar');
 	}
-}
+};
 
 // This utility is based on jsrsasign.X509.getSubjectString() implementation
 // we can not use that method directly because it requires calling readCertPEM()
 // first which as of jsrsasign@6.2.3 always assumes RSA based certificates and
 // fails to parse certs that includes ECDSA keys.
-function getSubjectCommonName(pem) {
+const getSubject = (pem) => {
 	const hex = jsrsasign.pemtohex(pem);
 	const d = ASN1HEX.getTLVbyList(hex, 0, [0, 5]);
-	const subject = X509.hex2dn(d); // format: '/C=US/ST=California/L=San Francisco/CN=Admin@org1.example.com/emailAddress=admin@org1.example.com'
+	return X509.hex2dn(d); // format: '/C=US/ST=California/L=San Francisco/CN=Admin@org1.example.com/emailAddress=admin@org1.example.com'
+};
+
+const getSubjectCommonName = (pem) => {
```

Comment:
I don't _think_ this is used anywhere. if not, might as well remove it. Just be careful to make sure the corresponding unit tests still drive `getSubject()` instead where appropriate.